spec/controllers/idv/review_controller_spec.rb

```diff
@@ -73,6 +80,53 @@ def show
     end
   end
 
+  describe '#confirm_current_password' do
+    controller do
+      before_action :confirm_current_password
+
+      def show
+        render text: 'Hello'
+      end
+    end
+
+    before(:each) do
+      stub_sign_in(user)
+      routes.draw do
+        post 'show' => 'idv/review#show'
+      end
+      allow(subject).to receive(:confirm_idv_steps_complete).and_return(true)
+      allow(subject).to receive(:confirm_idv_attempts_allowed).and_return(true)
+      idv_session.params = user_attrs.merge(phone_confirmed_at: Time.zone.now)
+      allow(subject).to receive(:idv_session).and_return(idv_session)
```

Comment:
I think this is a good example of why `before` and `let` statements are tricky. When reading this diff, I have no idea what `idv_session` is, am just trusting it's some thing (factory? mock? stub? string?)  above in the spec file ☘ 

If the test setup were all in each test, there'd be more duplication but I could see what was going on much more easily